omod-2.4/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/RestControllerTestUtils.java

```diff
@@ -9,9 +9,8 @@
  */
 package org.openmrs.module.webservices.rest.web.v1_0.controller;
```

Comment:
same thing with this class

Reply:
what is the same thing?

Reply:
It is an abstract class .

Reply:
did you do a search in github org:openmrs to see if any module implements these abstract test classes? if no module does we do not need to have a JUnit 4 and 5 version of them. We do need to figure this out first though